src/components/client.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Represents the client
+ * @type {module:eris.Client}
+ */
+const Eris = require('eris')
+const { Collection } = require('eris')
+module.exports = new Eris(process.env.BOT_TOKEN, {
+  messageLimit: 10
+})
+
+module.exports.wildbeastVoiceConnections = new Collection(require('../classes/VoiceConnection'))
+
+// patch event emitter to allow for our custom event structure
+module.exports._ogEmit = module.exports.emit
+module.exports.emit = function emit () {
+  this._anyListeners.forEach(listener => listener.apply(this, [arguments]))
+  return this._ogEmit.apply(this, arguments)
```

Comment:
Should we put some better variable names here? Something like this maybe:
```suggestion
module.exports._defaultEmit = module.exports.emit
module.exports.emit = function emit () {
  this._anyListeners.forEach(listener => listener.apply(this, [arguments]))
  return this._defaultEmit.apply(this, arguments)
```